dataland-community-manager/src/main/resources/application.properties

```diff
@@ -0,0 +1,37 @@
+springdoc.show-actuator=true
+springdoc.swagger-ui.oauth2RedirectUrl=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/community/swagger-ui/oauth2-redirect.html
+springdoc.swagger-ui.oauth.use-pkce-with-authorization-code-grant=true
+springdoc.swagger-ui.oauth.client-id=dataland-public
+management.endpoints.web.exposure.include=health, info
+management.endpoint.health.show-details=always
+management.info.git.mode=FULL
+server.tomcat.threads.min-spare=100
+server.shutdown=graceful
+server.servlet.context-path=/community
+spring.security.oauth2.resourceserver.jwt.issuer-uri=https://${PROXY_PRIMARY_URL:local-dev.dataland.com}/keycloak/realms/datalandsecurity
+spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://keycloak:8080/keycloak/realms/datalandsecurity/protocol/openid-connect/certs
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.username=community_manager
+spring.datasource.password=${COMMUNITY_MANAGER_DB_PASSWORD}
+spring.datasource.url=jdbc:postgresql://community-manager-db:5432/community_manager
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.show-sql=false
+spring.jpa.hibernate.ddl-auto=update
+spring.jackson.deserialization.fail-on-unknown-properties=true
+server.error.whitelabel.enabled=false
+spring.mvc.throw-exception-if-no-handler-found=true
+spring.web.resources.add-mappings=false
+spring.jpa.open-in-view=false
+spring.datasource.hikari.maximum-pool-size=50
+mailjet.api.url=https://api.eu.mailjet.com
+dataland.notification.sender.address=info@dataland.com
+dataland.notification.sender.name=Dataland
+dataland.notification.receivers.bulk.data.request=${NOTIFICATION_RECEIVERS_BULK_DATA_REQUEST:dummy.receiver@dataland.com}
```

Comment:
Are the values: dummy.receiver@dataland.com and dummy.receiver2@dataland.com correct or should those be changed to working emails?

Reply:
They are fallbacks if the env with the notification receivers is not set.
They have the right format, but are not real, so we don't bother someone with sending that person hundreds of mails while using the endpoint, e.g. on dev or during tests.

Reply:
ok